corehq/apps/users/models.py

```diff
@@ -1598,6 +1603,8 @@ class CommCareUser(CouchUser, SingleMembershipMixin, CommCareMobileContactMixin)
     user_location_id = StringProperty()
 
     is_anonymous = BooleanProperty(default=False)
+    login_attempts = IntegerProperty(default=0)
```

Comment:
Aren't these properties redundant with the `login_attempts` and `attempt_date` defined in `CouchUser`?

Reply:
Yeah, it was accidental commit addition. Will removed that